### ISSUE TITLE
Is there command line interface to control mimc?

### ISSUE THREAD

**Issue by diskotechjam** [MycroftAI/mimic#136]
I currently use mimic to read articles from my newsfeed, which works great, but occasionally, I need to pause playback when I am distracted by something else. Are there any command line interfaces to control mimic?

**Comment by forslund** [comment]
Not really. You could output to file and then play tge file with a media player.

**Comment by MaheshVelankar** [comment]
(For some reason, I am not able to open new issue here, so I had to add this here. I request you to kindly move it into a separate thread.
Thanks)

This is a question rather than an issue.

I wish to use this utility for Marathi language, for which I have my on parser to convert Unicode (utf8) Devanagari text into phonemes. 

I want to know how can use it? Which part of the code should I look into in order to modify it to use my parser instead of whatever parser this utility is using.

I have just started looking into this utility. So please excuse me if this is too trivial etc.

Thanks in advance
